__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+
+afterAll(() => db.end());
+
+describe("3. GET /api/categories", () => {
+    it("200: Should respond with an array of all the category objects", () => {
+        return request(app)
+            .get("/api/categories")
+            .expect(200)
+            .then(({ body: categories }) => {
+                const categoryArr = categories.categories;
+                expect(categoryArr).toHaveLength(4);
+                categoryArr.forEach((category) => {
+                    expect(category).toEqual(
+                        expect.objectContaining({
+                            slug: expect.any(String),
+                            description: expect.any(String)
+                        })
+                    );
+                });
+            });
+    });
+});
```

Comment:
Think about what error could affect this endpoint. What if a url is entered incorrectly?